src/ICSharpCode.SharpZipLib/BZip2/BZip2InputStream.cs

```diff
@@ -1,3 +1,7 @@
+#if !NETSTANDARD2_0 && !NETFRAMEWORK
```

Comment:
This effectively does the same thing, even if we are not targeting any .NET core ATM, it would still apply if we (or someone compiling from source) did.
Not sure that this is clearer, it explicitly states what versions are supported, instead of which ones isn't...
```suggestion
#if NETSTANDARD2_1_OR_GREATER || NETCOREAPP3_1_OR_GREATER
```